packages/commons-server/test/suites/templating-helpers/request-helpers.spec.ts

```diff
@@ -507,6 +518,16 @@ describe('Request helpers', () => {
       );
       expect(parseResult).to.be.equal('dolphin');
     });
+    it('should return the value of parameter with dots in parameter name', () => {
+      const parseResult = TemplateParser(
+        false,
+        "{{queryParam 'param1\\.name' undefined true}}",
```

Comment:
shouldn't it be a test with `queryParamRaw` instead?

Reply:
Damn! I missed that. Thanks for catching it. Sorry about oversight.

Reply:
No problem!

Reply:
There's same problem on test above it from where I copied. I will fix that as well.

Reply:
Ahah, indeed. Thanks for fixing my bad code!

Reply:
No problem!
Done. Hope everything is fine now :)

Reply:
Thanks, just in time for the release :)